account-kit/smart-contracts/src/msca/client/client.test.ts

```diff
@@ -165,6 +165,64 @@ describe("Modular Account Multi Owner Account Tests", async () => {
     `);
   }, 100000);
 
+  it("should update ownership successfully", async () => {
+    const provider = await givenConnectedProvider({
+      signer: signer2,
+      owners,
+    });
+    await setBalance(instance.getClient(), {
+      address: provider.getAddress(),
+      value: parseEther("1000"),
+    });
+    const signer3 = LocalAccountSigner.mnemonicToAccountSigner(
+      MODULAR_MULTIOWNER_ACCOUNT_OWNER_MNEMONIC,
+      { accountIndex: 2 }
+    );
+
+    // Deploy account and update owners. Remove signer1 and add signer3.
+    let result = await provider.updateOwners({
+      args: [[await signer3.getAddress()], [await signer1.getAddress()]],
```

Comment:
is signer1 already on the account? or is this part of the test -- it should not revert if a signer is not on the account?

I ask because you attach `signer2` above to this and wanted to make sure this was intentional

Reply:
`signer1`  and `signer2` are the initial owners. So here we deploy, set the initial owners, and update them at the same time. This one doesn't revert.